charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -298,3 +298,7 @@ spec:
                   pattern: '^(\d+(e\d+)?|\d+(\.\d+)?(e\d+)?[EPTGMK]i?)$'
                 scalyr_server_url:
                   type: string
+        status:
+          type: object
+          additionalProperties:
+            type: string
```

Comment:
Just to start the flamewar in yet another Github repo. You are missing newlines in the end of some of your files. 😈

Reply:
Thanks. Used my private Laptop whose editor lacks the go linting.